cli/setup.mjs

```diff
@@ -123,6 +123,14 @@ export async function setupDocs(docsDir, opts = {}) {
     routeRules: {
       ...Object.fromEntries(Object.entries(docsconfig.redirects || {}).map(([from, to]) => [from, { redirect: to }])),
     },
+    icon: {
+      customCollections: [
+        {
+          prefix: 'custom',
+          dir: resolve(docsDir, '.docs/assets/icons'),
```

Comment:
1. Would it work without warn if dir does not exists?
2. Can we stack multiple custom collections? (e.g, undocs can have a built-in preset of frameworks like h3 to render in other docs)
3. What you think about simpler `.docs/icons`?